app/src/main/res/values/strings.xml

```diff
@@ -125,4 +125,7 @@
     <string name="title_side_right_photo">Right</string>
     <string name="title_side_behind_photo">Behind</string>
     <string name="title_side_front_up_photo">Front up</string>
+    <string name="title_bill_label">Sale amount</string>
+    <string name="title_bill_description">Bill</string>
+    <string name="title_total">Total</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.title_total` appears to be unused
 Unused resources make applications larger and slow down builds.